relnotes/pkg-iteration.feature.md

```diff
@@ -0,0 +1,15 @@
+## The package iteration can now be specified via `PKGITERATION`
+
+* `Makd.mak`
```

Comment:
Don't make this a list, the new format suggest just a h2 title and then regular paragraphs.